bot/exts/utils/utils.py

```diff
@@ -206,6 +207,75 @@ async def vote(self, ctx: Context, title: clean_content(fix_channel_mentions=Tru
         for reaction in options:
             await message.add_reaction(reaction)
 
+    @checks.cooldown_with_role_bypass(1, 60, BucketType.user, bypass_roles=STAFF_ROLES)
```

Comment:
What use is there allowing normal users to use this command?

Reply:
It's a useful utility for moderation, but there is no extra development or running costs to allow users to run it. I added the cooldown check to avoid spam, but honestly, it uses an unlimited resource, so we don't even need that. I think it's a nice feature to have.